app/kumactl/cmd/apply/apply_test.go

```diff
@@ -178,6 +182,40 @@ var _ = Describe("kumactl apply", func() {
 		Expect(resource.Meta.GetNamespace()).To(Equal("default"))
 	})
 
+	It("should apply a new Dataplane resource from URL", func() {
+		// setup http server
+		mux := http.NewServeMux()
+		server := httptest.NewServer(mux)
```

Comment:
You need to stop server in the end of the test, e.g.

```
server := httptest.NewServer(mux)
defer server.Close()
```

Otherwise, it keeps hanging around and consume resources

Reply:
done